Tribler/community/multichain/database.py

```diff
@@ -145,6 +145,9 @@ def get_by_hash(self, hash):
         :param hash: The hash of the block that needs to be retrieved.
         :return: The block that was requested or None
         """
+        if hash is None:
```

Comment:
Why is this checked here instead of avoiding calling this function at all if there's no hash?